### ISSUE TITLE
norm_pix_loss question

### ISSUE THREAD

**Issue by surajraj99** [parthagrawal02/MAE_GAN#2]
For this implementation, shouldn't norm_pix_loss be False. If you were to have it on, I would imagine the discriminator would have a very easy time discriminating between patches from the original image and those that are generated from the decoder. It seems though, you have norm_pix_loss as True by default. Could you provide insight?

Also, as an aside, why do you cast things to double? Like in line 232 of models_mae.py and line 160 of mae_pretrain.py?

Thanks!

**Comment by parthagrawal02** [comment]
Why? can you explain more?

**Comment by surajraj99** [comment]
For the norm_pix_loss: aren't the generated patches going to have different range of values compared to the original image? Also if you now add the generated patches to the original image in place of the masked portions, won't you get something weird like shown in this github issue: https://github.com/facebookresearch/mae/issues/12.

Also in the paper you referenced, it seems like the reconstructions they have don't use the norm_pix_loss = True, because the images in Figure 3 don't have those discontinuous edges around patches characteristic of norm_pix_loss being True.

**Comment by parthagrawal02** [comment]
Okay, understood, I had my signals normalised during preprocessing, hence didn't give it a thought.
